src/app/pages/news-listing/news-listing.component.ts

```diff
@@ -0,0 +1,15 @@
+import { ChangeDetectionStrategy, Component, OnInit } from '@angular/core';
+import { compareAsc, compareDesc, isAfter, parse } from 'date-fns';
+import { getOrderedNews } from 'src/app/utils/get-ordered-news.utils';
+import { newsItems } from 'src/content/news/news-directory';
+
+@Component({
+  templateUrl: 'news-listing.component.html',
+  styleUrls: ['./news-listing.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class NewsListingComponent {
+  readonly allowedPrefix = 'news/';
```

Comment:
I don't think this gets used. Also a lot of unused imports in this file.

Reply:
Fixed, looks like something broke and linter is not running during commits.